### DISCUSSION TITLE
Why does "brew info ansible" show "stable 3.0.0" when the latest stable release is 2.10.6?

### DISCUSSION THREAD

**Discussion by wemove-wr** [Homebrew/discussions#1031] [Everyday usage] [answered]
Hi together,

i am confused about the version information for ansible:

Here:
```
brew info ansible               
ansible: stable 3.0.0 (bottled), HEAD
Automate deployment, configuration, and upgrading
https://www.ansible.com/
/usr/local/Cellar/ansible/3.0.0 (39,982 files, 373.2MB) *
  Poured from bottle on 2021-02-23 at 12:06:39
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/ansible.rb
License: GPL-3.0-or-later
```

And then:
```
which ansible
/usr/local/bin/ansible

/usr/local/bin/ansible --version
ansible 2.10.6
  config file = /Users/wrheinschmitt/.ansible.cfg
  configured module search path = ['/Users/wrheinschmitt/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/3.0.0/libexec/lib/python3.9/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.9.2 (default, Feb 24 2021, 13:26:09) [Clang 12.0.0 (clang-1200.0.32.29)]
```

2.10.6 is the latest stable release:
https://github.com/ansible/ansible/tags

Why does "brew info ansible" show "stable 3.0.0" when the latest stable release is 2.10.6?

Greetings to all

**Comment by Bo98** [comment, marked as answer]
There is a difference between the version of [`ansible-base`](https://pypi.org/project/ansible-base/) (which `ansible --version` shows) and the version of the [`ansible`](https://pypi.org/project/ansible/) CLI. Future versions of `ansible` will clarify this: https://github.com/ansible/ansible/pull/72287

**Reply by wemove-wr** [reply]
Thanks Bo!

For all those interested here matching articles:

Thoughts on Restructuring the Ansible Project (July 23, 2019)
https://www.ansible.com/blog/thoughts-on-restructuring-the-ansible-project

5 Things to Know About Ansible 2.10 (October 27, 2020)
https://www.wwt.com/article/5-things-to-know-about-ansible-2-10

Ansible 3.0.0 Q&A (February 18, 2021)
https://www.ansible.com/blog/ansible-3.0.0-qa
